### PR TITLE
Hue integration tests

### PR DESCRIPTION
**Pull request by adam-stasiak**
These scripts can be used for testing Hue init action on single, standard and ha configurations.

### CODE REVIEW

File: hue/create.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```
Comment (karth295) [inline hue/create.sh]: Is this file ever used? I think it needs to be deleted

File: hue/verify_hue_running.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+import requests
+import socket
+import subprocess
+
+BASE = 'localhost'
+PORT = 8888
+
+
+class HueApi(object):
```
Comment (karth295) [inline hue/verify_hue_running.py]: Similar feedback on the design of this class as in GangliaApi -- getter methods should not mutate the state of the object. Consider making these *Api classes stateless, and make assertions in other functions

**Comment by adam-stasiak** [comment]
Review changes applied.

**Comment by adam-stasiak** [comment]
@karth295 @functicons Could you take a look on this please?

**Comment by adam-stasiak** [comment]
@karth295 All changes are applied. Could you check this once again?

**Pull request merged**